README.md

```diff
@@ -72,15 +88,23 @@ Apply this network definition to the k8s cluster:
 After a minute or two you should have a functioning Ceramic network.
 Check the status of the network:
 
-    kubectl describe network small
+    kubectl describe network <initials>-small
+
+Keramik places each network into its own namespace named after the name of the network. You can default your context
+to this namespace using:
+
+    kubectl config set-context --current --namespace=keramik-<initials>-small
 
-Keramik places each network into its own namespace named after the name of the network.
 Inspect the pods within the network using:
 
-    kubectl -n keramik-small get pods
+    kubectl get pods
 
 >HINT: Use tools like [kubectx](https://github.com/ahmetb/kubectx) or [kubie](https://github.com/sbstp/kubie) to work with multiple namespaces and contexts.
 
+When you're finished, you can teardown your network with the following command:
+
+    kubectl delete namespace keramik-<initials>-small
```

Comment:
This should be `kubectl delete network <initials>-small` Otherwise the operator will recreate the namespace.